app/views/items/show.html.erb

```diff
@@ -5,9 +5,17 @@
 
     <div class="carousel-inner">
       <div class="carousel-item active">
-        <img class="d-block w-100" src="https://images.unsplash.com/photo-1512621776951-a57141f2eefd?crop=entropy&cs=tinysrgb&fm=jpg&ixlib=rb-1.2.1&q=80&raw_url=true&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1770" alt="First slide">
```

Comment:
@valerow-creator Could you put this line as a comment instead of removing it? Just in case during the intermediate demo we need to hardcode the picture ourselves. Thanks!